src/config.ts

```diff
@@ -89,6 +111,28 @@ function validateConfig(config: ConsentManagerConfig): boolean {
     errors.push('Consent manager UI config missing!');
   }
 
+  const dismissedViewStateParam = 'dismissedViewState';
+  const initialViewStateByPrivacyRegimeParam =
+    'initialViewStateByPrivacyRegime';
+  const {
+    [dismissedViewStateParam]: dismissedViewState,
+    [initialViewStateByPrivacyRegimeParam]: initialViewStateByPrivacyRegime,
+  } = config;
+  if (dismissedViewState) {
+    validateViewState(dismissedViewState, dismissedViewStateParam, errors);
+  }
+  if (initialViewStateByPrivacyRegime) {
+    Object.values(initialViewStateByPrivacyRegime).forEach((viewState) => {
+      if (viewState) {
+        validateViewState(
+          viewState,
+          `${initialViewStateByPrivacyRegimeParam} map value`,
```

Comment:
what's the `map value` ?

Reply:
`initialViewStateByPrivacyRegime` is a map of `PrivacyRegime`s to `ViewState`s. We are validating each of the map values in this structure